cluster/gce/config-default.sh

```diff
@@ -95,6 +95,8 @@ export KUBELET_TEST_ARGS=${KUBE_KUBELET_EXTRA_ARGS:-}
 
 # Default container runtime
 export CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-containerd}
+# Default container runtime for windows
+export WINDOWS_CONTAINER_RUNTIME=${KUBE_WINDOWS_CONTAINER_RUNTIME:-docker}
```

Comment:
Is there any other runtime that works in windows?

Reply:
containerd is Alpha for Windows and may be marked Beta for k8s 1.19. It still has some functional and performance limitations, so we don't want it to be our default yet.